packages/jira-adapter/src/adapter.ts

```diff
@@ -129,7 +133,7 @@ export default class JiraAdapter implements AdapterOperations {
         client,
         paginator,
         config,
-      }, filterCreators)
+      }, filterCreators, objects.concatObjects)
```

Comment:
nit:
```suggestion
      },
      filterCreators,
      objects.concatObjects,
    )
```
(and also before the first `{`)